src/output.rs

```diff
@@ -52,7 +52,9 @@ impl OutputAssertion {
         } else {
             self.matches_exact(&observed)
         };
-        result.map_err(|e| self.kind.map_err(e, cmd))
+        result.map_err(|e| super::errors::ErrorKind::OutputMismatch(cmd.to_vec(), e, self.kind))?;
```

Comment:
Isnt't ErrorKind in scope from the glob import?

Reply:
The `ErrorKind` that is in scope is from `self`, not `super`.